next-auth.d.ts

```diff
@@ -26,6 +26,8 @@ declare module 'next-auth' {
 		 * Object representing an authenticated user.
 		 */
 		user: {
+			/** The user's HashiCorp ID */
```

Comment:
💭  Not actually sure this is correct, but taking a shot at fleshing out this comment to try to clarify my own understanding!

```suggestion
			/** The user's HashiCorp Cloud IDP ID. Optional, as some users may still have pre-Cloud-IDP accounts. */
```

Reply:
@zchsh (PR auto merged before I could respond here) 

HashiCorp "ID" and "Cloud IDP ID" are largely the same, IMO.

"ID" is a more generic label, whereas "Cloud IDP ID" includes the exact service name. I reached for the generic label and didn't give much deep thought after the fact.